### PR TITLE
copyright for image assets

### PR DESCRIPTION
**Pull request by tessi**
image copyrights
- remove unused images
- move webalys images into a separate subfolder and add a license for them

see: https://www.openproject.org/issues/2070

### CODE REVIEW

File: app/assets/images/webalys/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# All images in this directory are lincensed under the Creative Commons License
```
Comment (apfelfabrik) [inline app/assets/images/webalys/LICENSE.md:1]: http://creativecommons.org/choose/

**Comment by apfelfabrik** [comment]
I know this is a rather center part of the whole PR, but i would argue that webalys is not the correct name for the image folder you're introducing. It's the company name, most likely trademarked, yada yada. I believe "minicons" is a better name for this.

**Comment by apfelfabrik** [comment]
I have removed the executable bit on the remaining images. That always annoyed me. Hope you don't mind, boyscout and stuff.

**Comment by tessi** [comment]
we have both: minicons icons and images from his GUI framework. so I thought the common name between those two is the name of his page. 

:thumbsup: for the `chmod -x`

Your're right, the link to CC is missing. I'll add that on my next finn day (if noone is fixing that earlier)

**Comment by apfelfabrik** [comment]
What I meant is, it doesn't say which CC license. There are plenty, and no "the CC license". We should either specify the exact name of the license, or remove the statement. I can do either, just tell me which way to go.

**Comment by apfelfabrik** [comment]
> we have both: minicons icons and images from his GUI framework. so I thought the common name between those two is the name of his page.

Fair enough.

**Comment by tessi** [comment]
I'll cite from the webalys page(s) for the license thing:
- User Interface Framework by Webalys is licensed under a [Creative Commons Attribution 3.0 Unported License](https://creativecommons.org/licenses/by/3.0/). 
- Minicons Free Vector Icons Pack by Webalys is licensed under a [Creative Commons Attribution 3.0 Unported License](http://www.webalys.com/minicons/license.php).

For the GUI framework he directly links a CC license. For the minicons he uses the same license name, but links to his own page. Not sure if it's the same CC license.

**Comment by mfrister** [comment]
@tessi What's the status of this pull request? Do we need to decide whether we'll just use CC BY 3.0 although he links to his own license page?

I actually don't see the CC-BY 3.0 and link to his own license page as contractiction. On the [license page](http://www.webalys.com/minicons/license.php) he says:

> You're allowed to... Use the royalty-free icons for any personal, commercial project

So if there's a page saying they're under CC-BY 3.0, I think we're ok.

Btw, should we have a page in the UI giving attribution?

**Comment by tessi** [comment]
@meeee thanks for the reminder. I forgot to push/update the pull request. Also did a rebase, so this PR is mergable.

@apfelfabrik I added links to the original license texts.

**Comment by apfelfabrik** [comment]
I'm good. Merging this one. Thanks!

**Pull request merged**